cygnus-arcgis/src/main/java/com/telefonica/iot/cygnus/backends/arcgis/baselogger/BaseLogger.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * 
+ */
+package com.telefonica.iot.cygnus.backends.arcgis.baselogger;
+
+import org.apache.log4j.Logger;
+
+/**
+ * @author dmartinez
+ *
+ */
+public class BaseLogger implements BaseLoggerInterface {
+
+    static final Logger LOGGER = Logger.getLogger(BaseLogger.class);
```

Comment:
@dmartinezgomez can you elaborate a little bit more about your BaseLgger class. Sorry, I'm confused, about this. I understood that you modiffied the way Cygnus logs, but this is a completely new logger.

Reply:
This model was designed to be a PDI (kettle) plugin, and BasseLogger was the way to make logs work both Cygnus and PDI.  Finally we make an specific project for cygnus, and changed logs to work Cygnus-like.